generators/entity-gatling/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2013-2021 the original author or authors from the JHipster project.
+ *
+ * This file is part of the JHipster project, see https://www.jhipster.tech/
+ * for more information.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable consistent-return */
+const writeFiles = require('./files').writeFiles;
+const utils = require('../utils');
+const BaseBlueprintGenerator = require('../generator-base-blueprint');
+
+/* constants used throughout */
+let useBlueprints;
+
+module.exports = class extends BaseBlueprintGenerator {
+  constructor(args, opts) {
+    super(args, opts);
+    utils.copyObjectProps(this, opts.context);
```

Comment:
```suggestion
```

Should be done at default step, it not ready at this moment.
Like others generators:
https://github.com/jhipster/generator-jhipster/blob/766f67b1a234fae60f8e57b1d5bb82c64c519be4/generators/entity-server/index.js#L98-L102